src/main/java/Duke.java

```diff
@@ -1,10 +1,128 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    private static final String welcomeMessage = "Hello! I'm KJ\nHow can I help you today?";
+
+    private static final String completeMessage = "Nice! I've marked this task as done:";
+
+    private static final String byeMessage = "Bye. Hope to see you again soon!";
+
+    private static final String listMessage = "Here are the tasks in your list:";
+
+    private static final int MAXIMUM_TASKS = 100;
+
+    private static boolean shouldContinue;
+
+    private static String command;
+
+    private static String action;
+
+    private static Scanner input;
+
+    private static Task[] instructions;
+
+    private static int count;
```

Comment:
Consider being more concise with what this "count" variable is actually counting/tracking. An example could be "taskCount"

Reply:
You are right, it is rather unclear what count really signifies here. Thanks for pointing out!